assets/js/main.js

```diff
@@ -156,6 +156,32 @@ $(document).ready((function ($) {
         return false;
     });
 
+    $('.appointment-button').click(function () {
+        var $form = $('.appointment-form'),
+            data = getFormData($form),
+            host = 'http://127.0.0.1:8000/',
+            path = 'api/appointment/',
+            url = host + path;
+
+        $.ajax({
+            type: 'POST',
+            url: url,
+            data: JSON.stringify(data),
+            beforeSend: removeFieldErrors,
+            success: function () {
+                $('#section .alert-success').fadeIn(function () {
+                    $(this).fadeOut(14000);
+                });
+                $form.find('input').each(function () {
+                    $(this).val('');
+                    $(this).prop('checked', false);
+                });
+            },
+            error: showFieldErrors,
```

Comment:
### Part 1:

![Screenshot 2021-11-20 at 16 21 54](https://user-images.githubusercontent.com/12160621/142729728-748e5e39-922d-4a40-818f-10e46667d874.png)

If I submit an empty HTML form it highlights only two fields with a red outline.
This can be fixed by renaming input IDs in a way they match fields on a serializer in the backend:

![mroomapp – index html 2021-11-20 16-24-51](https://user-images.githubusercontent.com/12160621/142729918-bdbb02d6-31d4-48b0-b12f-e4fd6ac64fa3.png)

All because function `showFieldErrors` is looking for IDs using key values that are returned from the backend:

![mroomapp – main js 2021-11-20 16-27-57](https://user-images.githubusercontent.com/12160621/142729975-15a846a8-2f7a-48a9-b3a3-614f3e795ea0.png)

### Part 2:

Also, you need to show error message next to the field. Rename `validate` class to `invalid-feedback`:

![mroomapp – index html 2021-11-20 16-30-40](https://user-images.githubusercontent.com/12160621/142730093-cc01838b-3256-4278-9ec7-8ca99707e562.png)

![mroomapp – main js 2021-11-20 16-31-08](https://user-images.githubusercontent.com/12160621/142730099-ed01624c-d463-4bc4-b3e5-af67062c4e97.png)


#### Expected result:

![Screenshot 2021-11-20 at 16 33 17](https://user-images.githubusercontent.com/12160621/142730138-01f3ca11-a55f-4629-9a1c-fdf762cb1a20.png)